packages/core/src/server.ts

```diff
@@ -1014,14 +1028,22 @@ export class Controller extends API {
             members,
             groups,
             vaults,
+            directory,
         });
 
-        if (org.owner !== owner) {
-            await this.provisioner.orgOwnerChanged(
-                org,
-                org.getMember({ id: org.owner })!,
-                org.getMember({ id: owner })!
-            );
+        if (org.directory.syncProvider === "scim") {
```

Comment:
@MaKleSoft I initially implemented this in the frontend, but `directory` wasn't seeing `directory.scim` (even though it was properly being sent as serialized in the HTTP request). After a lot of struggling, I decided it wouldn't be bad to implement this in the backend, as if it was ever updated in some other way, it would not have to be a concern of the client, but it still doesn't work, somehow. The other settings work and get saved, my logging below sees the secret, but it's just not saved.